WcaOnRails/app/models/user.rb

```diff
@@ -68,6 +68,13 @@ def name_must_match_person_name
     end
   end
 
+  validate :dob_must_be_on_the_past
+  def dob_must_be_on_the_past
+    if dob.year >= Date.today.year
```

Comment:
`dob >= Date.today`? Today I could enter `2016-12-31` and I'd be about -340 days old.